spec/features/deserialization_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.feature "deserialization" do
+  include ActiveJobHelper
+  include RackTestFeaturesHelper
+
+  before do
+    Timecop.travel Time.new(2015, 9, 1, 15, 47).utc
+    Capybara.current_driver = :rack_test
+    VisitMetricsEntry.delete_all
+  end
+
+  after do
+    Timecop.return
+  end
+
+  let(:data) {
+    YAML.load(
+      File.read(
+        File.expand_path('../../fixtures/deserialization_data.yml', __FILE__)
+      )
+    )
+  }
+
```

Comment:
Minor style thing--the ruby-style guide prefers do...end for multiline blocks.  I'm not sure whether or no rubocop will alert on this...